src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Testing/MsTestV2/TestMethodAttributeExecuteIntegration.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.MsTestV2
+{
+    /// <summary>
+    /// Microsoft.VisualStudio.TestPlatform.TestFramework.Execute calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+        Assembly = "Microsoft.VisualStudio.TestPlatform.TestFramework",
+        Type = "Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute",
+        Method = "Execute",
+        ReturnTypeName = "Microsoft.VisualStudio.TestTools.UnitTesting.TestResult",
+        ParametersTypesNames = new[] { "Microsoft.VisualStudio.TestTools.UnitTesting.ITestMethod" },
+        MinimumVersion = "14.0.0",
+        MaximumVersion = "14.*.*",
+        IntegrationName = IntegrationName)]
+    public class TestMethodAttributeExecuteIntegration
+    {
+        private const string IntegrationName = "MSTestV2";
```

Comment:
Any reason you're not using the IntegrationIds enum?

Reply:
Mmm nope, I'll add it.

Reply:
done in f34a8cd